R/api.R

```diff
@@ -30,3 +30,41 @@ root <- function() {
   )
   lapply(versions, function(v) scalar(as.character(v)))
 }
+
+##' @porcelain GET /metadata => json(metadata)
+metadata <- function() {
+  # JIDEA-62: we will use relevant model version - from qs if specified, else
+  # from latest available metadata file
+  model_version <- scalar("0.1.0")
+  # JIDEA-62: we will read in correct metadata version according to
+  # model_version
+  metadata_file <- sprintf("metadata_%s.json", model_version)
+  response <- read_json(metadata_file)
```

Comment:
I was initially confused as to where this function was coming from, but now I see it's redefined in this package; I've added a suggestion to switch to `jsonlite::fromJSON()` instead